src/style.scss

```diff
@@ -0,0 +1,186 @@
+/* #region root */
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+
+:root {
+  --main-accent: #{$main-accent};
+  --secondary: #{$secondary};
+  --blue-accent: #{$blue-accent};
+  --white: #{$white};
+}
+
+body {
+  margin: 0;
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region card */
+
+.card {
```

Comment:
please use only 2 BEM blocks - 1 for stars, 1 for card, both of them should have their own files